drivers/net/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright (c) 2018 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+zephyr_sources_ifdef(CONFIG_ETH_RTT rtt.c)
```

Comment:
I am avoiding SLIP in the names, because it is not exactly SLIP over RTT. SLIP **protocol** does not transfer ethernet frames, but frames from upper level. Protocol that is implemented by this  driver is using just **encoding** defined by SLIP to encode ethernet frames. Additionally CRC-16 is calculated and added at the end of frame. See README for more details.

So if `rtt.c` isn't ok then maby `eth_rtt.c`?

Reply:
Still, the functionality is pretty similar to what `slip.c` in Zephyr does (with exception to the CRC part) - UART based ethernet interface with SLIP encoding. So I'd expect to follow the convention. But if you don't want to, `eth_rtt.c` should be fine as well.